addons/portal/models/ir_http.py

```diff
@@ -15,6 +14,6 @@ def _get_translation_frontend_modules_name(cls):
 
     @classmethod
     def _get_frontend_langs(cls):
-        if request and request.is_frontend:
+        if request and getattr(request, 'is_frontend', True):
```

Comment:
`_get_frontend_langs` is called during `http_routing.ir.http._match` where `is_frontend` is not set yet and that no backend endpoint matched. We have to assume *we are* going to match a frontend route, hence the default `True`. I should add a comment.